Formula/pyqt@5.rb

```diff
@@ -61,27 +63,38 @@ class PyqtAT5 < Formula
     sha256 "8bb1df553ba6a615f8ec3d9b9c5270db3e15e831a6161773dabfdc1a7afe4834"
   end
 
+  def pythons
+    deps.map(&:to_formula)
+        .select { |f| f.name.match?(/python@\d\.\d+/) }
+        .map(&:opt_bin)
+        .map { |bin| bin/"python3" }
+  end
+
   def install
-    site_packages = prefix/Language::Python.site_packages("python3")
     args = %W[
-      --target-dir #{site_packages}
       --scripts-dir #{bin}
       --confirm-license
       --no-designer-plugin
       --no-qml-plugin
     ]
-    system "sip-install", *args
+    components = %w[3d chart datavis networkauth purchasing webengine]
 
-    resource("PyQt5-sip").stage do
-      system "python3", *Language::Python.setup_install_args(prefix)
-    end
+    pythons.each do |python|
+      site_packages = prefix/Language::Python.site_packages(python)
+      args << "--target-dir" << site_packages
+      system "sip-install", *args
 
-    components = %w[3d chart datavis networkauth purchasing webengine]
-    components.each do |p|
-      resource(p).stage do
-        inreplace "pyproject.toml", "[tool.sip.project]",
-          "[tool.sip.project]\nsip-include-dirs = [\"#{site_packages}/PyQt#{version.major}/bindings\"]\n"
-        system "sip-install", "--target-dir", site_packages
+      resource("PyQt5-sip").stage do
+        system python, *Language::Python.setup_install_args(prefix),
+                       "--install-lib=#{prefix/Language::Python.site_packages(python)}"
+      end
+
+      components.each do |p|
+        resource(p).stage do
+          inreplace "pyproject.toml", "[tool.sip.project]",
+            "[tool.sip.project]\nsip-include-dirs = [\"#{site_packages}/PyQt#{version.major}/bindings\"]\n"
```

Comment:
```suggestion
          inreplace "pyproject.toml", "[tool.sip.project]", <<~EOS
            [tool.sip.project]
            sip-include-dirs = ["#{site_packages}/PyQt#{version.major}/bindings"]
          EOS
```
Something like this should work and is nicer to read.